pkg/controller/migration/convert/network_test.go

```diff
@@ -76,6 +77,7 @@ var _ = Describe("Convert network tests", func() {
 			}, operatorv1.PluginGKE),
 		)
 		It("should convert AWS CNI install", func() {
+			Skip("eks not supported until typha prometheus metrics support is added")
```

Comment:
@tmjd I wonder if we want to just ignore user's prometheus metrics settings for typha and disable them until we have a way to migrate them. Otherwise, we're knocking out all clusters installed with our aws-cni-policy-only  manifests

Reply:
I'd say that ignoring is probably the route to go for right now but we should make it a priority to add that capability.